packages/components-react/projects/react-ssr-wrapper/src/splitChildren.ts

```diff
@@ -0,0 +1,18 @@
+import type { ReactElement } from 'react';
+
+export const splitChildren = (
+  children: ReactElement | ReactElement[]
+): {
+  children: ReactElement[];
```

Comment:
All these types are actually not correct because of the parameters type
```
type ReactNode = ReactElement | string | number | ReactFragment | ReactPortal | boolean | null | undefined;
```
Since we don't do any modifications on the elements, a `number` stays a `number` and does not become a 
```
  interface ReactElement<P = any, T extends string | JSXElementConstructor<any> = string | JSXElementConstructor<any>> {
        type: T;
        props: P;
        key: Key | null;
    }
```.